docs/eso/eso.rst

```diff
@@ -4,6 +4,34 @@
 ESO Queries (`astroquery.eso`)
 ******************************
 
+.. warning::
+
+    Backward Compatibility Notice
+    ==============================
+
+    **The WDB (Web DataBase) API is being deprecated and replaced by TAP (Table Access Protocol)**,
+    a standardized interface for querying astronomical datasets using ADQL (Astronomical Data Query Language).
+    While the Python interface remains the same, the values accepted by the ``columns`` and ``column_filters``
+    parameters must reflect TAP field names and ADQL syntax. This means that,
+    although the structure of your code won't need to change, **the values you pass to the arguments**
+    ``columns`` **and** ``column_filters`` **must be revised** to comply with the new format.
```

Comment:
Since this text is about 'new' and 'change' etc., let's add a date tag.  Somewhere in here, say "as of May 2025".

Is WDB itself being taken offline?  It also helps to indicate that so that users know there is not some alternative other approach.

Reply:
(this is partly so that we don't have to figure out when to remove the "new" qualifier in this text; it's easy to forget to update documentation)

Reply:
True, good point 👌